docs/development/shareable-configs.md

```diff
@@ -50,12 +50,40 @@ If you use a non-scoped config, you must use a preset name!
+| GitHub with preset name            | `github>abc/foo:xyz`       | `xyz`     | `https://github.com/abc/foo` | `xyz.json`     | Default branch |
+| GitHub default with a tag          | `github>abc/foo#1.5.4`     | `default` | `https://github.com/abc/foo` | `default.json` | `1.5.4`        |
+| GitHub with preset name with a tag | `github>abc/foo:xyz#1.5.4` | `xyz`     | `https://github.com/abc/foo` | `xyz.json`     | `1.5.4`        |
+
+#### GitLab
+
+| name                               | example use                | preset    | resolves as                  | filename       | Git tag        |
+| ---------------------------------- | -------------------------- | --------- | ---------------------------- | -------------- | -------------- |
+| GitLab default                     | `gitlab>abc/foo`           | `default` | `https://gitlab.com/abc/foo` | `default.json` | Default branch |
+| GitLab with preset name            | `gitlab>abc/foo:xyz`       | `xyz`     | `https://gitlab.com/abc/foo` | `xyz.json`     | Default branch |
+| GitLab default with a tag          | `gitlab>abc/foo#1.5.4`     | `default` | `https://gitlab.com/abc/foo` | `default.json` | `1.5.4`        |
+| GitLab with preset name with a tag | `gitlab>abc/foo:xyz#1.5.4` | `xyz`     | `https://gitlab.com/abc/foo` | `xyz.json`     | `1.5.4`        |
+
+#### Gitea
+
+| name                              | example use               | preset    | resolves as                 | filename       | Git tag        |
+| --------------------------------- | ------------------------- | --------- | --------------------------- | -------------- | -------------- |
+| Gitea default                     | `gitea>abc/foo`           | `default` | `https://gitea.com/abc/foo` | `default.json` | Default branch |
+| Gitea with preset name            | `gitea>abc/foo:xyz`       | `xyz`     | `https://gitea.com/abc/foo` | `xyz.json`     | Default branch |
+| Gitea default with a tag          | `gitea>abc/foo#1.5.4`     | `default` | `https://gitea.com/abc/foo` | `default.json` | `1.5.4`        |
+| Gitea with preset name with a tag | `gitea>abc/foo:xyz#1.5.4` | `xyz`     | `https://gitea.com/abc/foo` | `xyz.json`     | `1.5.4`        |
+
+#### Self-hosted Git
+
+| name                              | example use                     | preset    | resolves as                          | filename        | Git tag        |
+| --------------------------------- | ------------------------------- | --------- | ------------------------------------ | --------------- | -------------- |
+| Local default                     | `local>abc/foo`                 | `default` | `https://github.company.com/abc/foo` | `default.json`  | Default branch |
+| Local with preset path            | `local>abc/foo//path/xyz`       | `default` | `https://github.company.com/abc/foo` | `path/xyz.json` | Default branch |
```

Comment:
Just asking to be sure, do you need the extra `/` in the example use colum when you're setting a path?

I'm talking about sections that look like this:

```
`local>abc/foo//path/xyz`
```

Reply:
You're right asking, that was not good and I found a bug while retesting this :sweat_smile: the last commit fixes all this :)